Source/JavaScriptCore/API/tests/MultithreadedMultiVMExecutionTest.cpp

```diff
@@ -51,13 +51,17 @@ void startMultithreadedMultiVMExecutionTest()
     WTF::initializeMainThread();
     JSC::initialize();
 
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+
 #define CHECK(condition, threadNumber, count, message) do { \
         if (!condition) { \
             printf("FAIL: MultithreadedMultiVMExecutionTest: %d %d %s\n", threadNumber, count, message); \
             failuresFound++; \
         } \
     } while (false)
 
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
```

Comment:
I'm surprising doing this around a `#define` does anything. I'd expected we either need to do it at the call sites or *within* the `CHECK` macro.

Reply:
Hmmm... I'm surprised too. I'll move it inside the macro.